config/rbac/metrics_role_binding.yaml

```diff
@@ -9,5 +9,5 @@ roleRef:
   name: compliance-operator-metrics
 subjects:
   - kind: ServiceAccount
-    name: prometheus-k8s
```

Comment:
I suppose we could keep this as is, right? Or does the name of the SA also have to change as part of the dependency bump?